integration-tests/testconfig/README.md

```diff
@@ -116,6 +116,83 @@ When processing TOML files, the system initially searches for a general (unnamed
 
 Finally `default.toml` file is envisioned to contain fundamental and universally applicable settings, such as logging configurations.
 
+### Chainlink Node TOML config
+
+To set custom config for Chainlink Node use `NodeConfig.BaseConfigTOML` in TOML. Example:
+
+```
+[NodeConfig]
+BaseConfigTOML = """
+[Feature]
+FeedsManager = true
+LogPoller = true
+UICSAKeys = true
+
+[Log]
+Level = 'debug'
+JSONConsole = true
+
+[Log.File]
+MaxSize = '0b'
+
+[OCR]
+Enabled = true
+DefaultTransactionQueueDepth = 0
+"""
+```
+
+To set base config for EVM network use `NodeConfig.CommonChainConfigTOML`. Example:
```

Comment:
Same here and I would say that this is the default chain-specific config that is used, when no custom one matches selected network

Reply:
@Tofel pls check 04cd351581d6d3c23f9d9959e8854f969d4e4083